From 9f1ebd03d6c4deebbd4d62d061ceeea4eec9be6e Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Mon, 2 May 2022 17:18:35 +0300
Subject: [PATCH 12/21] cmd: tlv_eeprom: add my copyright

While each of the previous individual changes is small, accumulated they
amount to a substantial effort. Explicitly add a Copyright declaration.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 cmd/tlv_eeprom.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmd/tlv_eeprom.c b/cmd/tlv_eeprom.c
index c110927cb5..c66116b2c4 100644
--- a/cmd/tlv_eeprom.c
+++ b/cmd/tlv_eeprom.c
@@ -7,6 +7,7 @@
  * Copyright (C) 2014 Srideep <srideep_devireddy@dell.com>
  * Copyright (C) 2013 Miles Tseng <miles_tseng@accton.com>
  * Copyright (C) 2014,2016 david_yang <david_yang@accton.com>
+ * Copyright (C) 2022 Josua Mayer <josua@solid-run.com>
  */
 
 #include <common.h>
-- 
2.40.1

